library/ssl_tls.c

```diff
@@ -6881,19 +7071,59 @@ size_t mbedtls_ssl_get_max_frag_len( const mbedtls_ssl_context *ssl )
      */
     max_len = ssl_mfl_code_to_length( ssl->conf->mfl_code );
 
-    /*
-     * Check if a smaller max length was negotiated
-     */
+    /* Check if a smaller max length was negotiated */
     if( ssl->session_out != NULL &&
```

Comment:
Some tests fail here because of #1941.

Reply:
I believe this was fixed by basing this PR on #1942.